### PR TITLE
[IMP] enterprice-mx: Active the Journal's check box to allow cancel invoice from demo

### PR DESCRIPTION
**Pull request by JesusZapata**
Active the Journal's check box to allow cancel invoice from demo
Referring #10

### CODE REVIEW

File: l10n_mx_enterprise/demo/global.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-
+    Configuration
+-
+
+-
+    !python {model: account.journal}: |
+        self.env['account.journal'].search([
+            ['name', '=', 'Customer Invoices'],
```
Comment (luistorresm) [inline l10n_mx_enterprise/demo/global.yml]: @JesusZapata 

Could you use _('Customer Invoices') to ensure that work on instances in spanish?

Regards

Comment (JesusZapata) [inline l10n_mx_enterprise/demo/global.yml]: Done, I was not sure if an import could be made!

**Comment by moylop260** [comment]
We have currently a red status in branch stable but if see the runbot build is a warning (yellow).
But the build of this PR has a error (red).
Could you review it?

**Comment by moylop260** [comment]
Add a print screen using runbot where this value is enabled

**Comment by JesusZapata** [comment]
Screenshot of check allow cancel invoice from demo
image

The warning is 
```
2017-01-10 20:14:12,481 179 INFO openerp_template odoo.modules.loading: loading l10n_mx_reports/demo/account_invoice_demo.xml
2017-01-10 20:14:13,973 179 INFO openerp_template odoo.modules.loading: loading l10n_mx_reports/demo/res_partner_demo.xml
2017-01-10 20:14:13,985 179 WARNING openerp_template odoo.models: res.partner.write() with unknown fields: type_of_operation
2017-01-10 20:14:14,281 179 WARNING openerp_template odoo.models: res.partner.write() with unknown fields: type_of_operation
2017-01-10 20:14:14,517 179 WARNING openerp_template odoo.models: res.partner.write() with unknown fields: type_of_operation
2017-01-10 20:14:14,854 179 WARNING openerp_template odoo.models: res.partner.write() with unknown fields: type_of_operation
```

And error is 
```
2017-01-10 20:14:51,508 216 INFO openerp_test odoo.modules.registry: module l10n_mx_enterprise: creating or updating database tables
2017-01-10 20:14:51,558 216 INFO openerp_test odoo.modules.loading: loading l10n_mx_enterprise/demo/global.xml
2017-01-10 20:14:52,295 216 INFO openerp_test odoo.modules.loading: loading l10n_mx_enterprise/demo/global.yml
2017-01-10 20:14:52,300 216 ERROR openerp_test odoo.tools.yaml_import: You have an empty block in your tests.
2017-01-10 20:14:52,352 216 INFO openerp_test odoo.modules.loading: 52 modules loaded in 14.85s, 5199 queries
2017-01-10 20:14:53,018 216 ERROR openerp_test odoo.modules.loading: At least one test failed when loading the modules.
2017-01-10 20:14:53,024 216 INFO openerp_test odoo.modules.loading: All post-tested in 0.00s, 0 queries
2017-01-10 20:14:53,026 216 INFO openerp_test odoo.models: Computing parent left and right for table ir_ui_menu...
```
I am going fix error on l10n_mx_enterprise

**Comment by JesusZapata** [comment]
I fixed the error in yml file!

**Comment by JesusZapata** [comment]
The warning Seems to be in the module l10n_mx_reports! 
I can register an issue at https://github.com/Vauxoo/mexico and solve it!
@moylop260 I can register that issue?

**Pull request merged**